mon-pix/tests/acceptance/user-dashboard-test.js

```diff
@@ -178,20 +178,78 @@ describe('Acceptance | User dashboard page', function() {
 
   describe('new dashboard information', function() {
```

Comment:
**suggestion(blocking):** Réordonner les tests autour d'un describe de test pour `{{#if this.hasNewInformationToShow}}` et ajouter éventuellement les cas de tests associés pour vérifier la table de vérité des conditions imbriquées